.gitignore

```diff
@@ -22,7 +22,9 @@ habitat/results
 /.direnv
 /.envrc
 results/
-
+.vagrant/
```

Comment:
Yes definitely.  My intent was that my changes to `.gitignore` would themselves be ignored, but it didn't seem to work out that way.  I'll have a go at the unit tests as well.